docs/guides/upgrading-v3.md

```diff
@@ -196,6 +196,31 @@ That allowed us to make the KMS configuration consistent with all other AWS reso
 
 `alexaSkill` events now require an `appId` ([learn more](../deprecations.md#support-for-alexaskill-event-without-appid-is-to-be-removed)). That change was required to implement a more stable deployment, as well as to deploy more restricted IAM permissions.
 
+### Lambda Hashing Algorithm
+
+By default, Lambda version hashes will now be generated using improved algorithm (fixed determinism issues). As it is a breaking change that requires more manual effort during migration, it is still possible (but not recommended) to keep using old algorithm by using the following configuration:
+
+```
+provider:
+  lambdaHashingVersion: 20200924
+```
+
+However, we highly encourage an upgrade to the new algorithm. Safe upgrade requires change in configuration or code of all your functions. You can do it safely following the guide below:
+
+**NOTE**: Please keep in mind that these changes require two deployments with manual configuration adjustment between them. It also creates two additional versions and temporarily overrides descriptions of your functions. Migration will need to be done separately for each of your environments/stages.
+
+1. Run `sls deploy` with additional `--enforce-hash-update` flag to override description for all your functions and force creation of new versions.
```

Comment:
```suggestion
1. Run `sls deploy` with additional `--enforce-hash-update` flag: that flag will override the description for all Lambda functions, which will force the creation of new versions.
```

I would also state clearly that we start from a Serverless v2 application. Maybe it's a bit over the top to explicitly write that? But at least as a user it's 100% clear. Something like this:

```
1. In a Serverless Framework v2 project
```

(or maybe put that part in the paragraph above?)

Reply:
I think we can add it, but I think it's also a bit redundant since this guide is about migrating from v2 to v3 specifically. Where would you exactly see a good place to add such information?

Reply:
👍  no problem then ignore that part